### ISSUE TITLE
Change management for Cerner cutover

### ISSUE THREAD

**Issue by ContentBrewmaster** [department-of-veterans-affairs/va.gov-cms#9338]
The change represented by the [epic 7158](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/7158) will let editors see whether a system is set to VISTA or Cerner.

[Cerner email ](https://docs.google.com/document/d/1pkwWH8fTyOq10jQuZgKTWFy9lchPwQB9Z4py8c-6ih4/edit)(for delivery by VHA Digital Media)
 
**1. Describe the change from the perspective of VA employees, in plain language. What is the employee going to experience with this change?**
Editors can see the source of truth for Veteran-facing web-based services, such as appointment creation.  

<img width="839" alt="image" src="https://user-images.githubusercontent.com/92043982/172253629-0fbb2c93-2407-49e4-970f-27cc8e017bac.png">

Ui guidance
![image](https://user-images.githubusercontent.com/92043982/172714536-b7e8a864-8fe8-4e26-8c7a-829967a1e8c7.png)

**2. Explain why the change is necessary from a CMS/platform and project goal perspective. Link to screenshots if that helps tell the story.**
The new EHR connects VA’s medical centers and clinics with the Department of Defense (DOD) and the U.S. Coast Guard. A single EHR gives providers access to more complete information regarding a patient’s medical history and helps create a more seamless health care experience for service members transitioning from active duty to Veteran status. 

There is a front end change for authenticated users using 5 React widgets:
- refill prescription
- make appointment
- get VA medical records
- send secure message
- view lab or test results

**3. Select the role(s) impacted by the change.**
- [ ] All CMS users
- [ ] All CMS editors
- [ ] Vet Center editor
- [x] VA Medical Center editor
- [ ] VA Outreach Hub editor
- [ ] VA Benefits Hub editor
- [ ] Campaign landing page editor
- [ ] VA Resources and Support editor
- [ ] Offices editor
- [ ] Supporting roles such as CMS account admin, Facilities or Public websites team member

**4. How will success be defined for this change?**
There's no confusion about the cutover to Cerner.

**5. How will success be measured for this change (audit, survey, other types of research, etc.)?**
No Help Desk tickets are generated when a facility switches over to Cerner.

**6. Does the change require new or updated:**
- [x] Knowledge Base article (new) for content admins only: [Electronic health records (EHR) system changes](https://prod.cms.va.gov/electronic-health-records-ehr-system-changes)

**7. Will the change be tested with users before release?**. 
- [x] Yes (for initial cutover only)
- [ ] No

**8. Are there any known dependencies?**
For a given cut-over, there is a go-no go directive coming from OCTO Digital Experience.

**Comment by jilladams** [comment]
@ContentBrewmaster two upcoming items here that Dave may have already mentioned: 
1. The Drupal source of truth work will go live on Tues 7/19 in the afternoon
2. Boise will cut over to Cerner on 7/23 for medical records, and on the morning of 7/25 for these listed widgets (via #8802).

**Comment by jilladams** [comment]
The scope of this ticket has gotten muddy for me, and I'd like to chat with you @ContentBrewmaster to clarify, when you have time this week. Some notes: 

1. Boise cutover was moved to 2023. 
2. The Drupal source of truth for the 5 widgets listed in this ticket body has been updated as of Sprint 63. Those 5 widgets now reference the listed Drupal EHR field to determine if a VAMC System is Cerner / VistA.
3. We will eventually EOL the hard-coded data method altogether (~EOY). I cut a separate ticket for CM related to that bigger picture change (before I remembered this ticket existed): https://github.com/department-of-veterans-affairs/va.gov-cms/issues/10022.

I do have an open ticket re: updating our runbooks with Change Management notes when a facility cuts over to Cerner: https://github.com/department-of-veterans-affairs/va.gov-cms/issues/9340. I'd also like to review that with you. Happy to schedule, if that's best, or let me know if you'd prefer to schedule when it suits your calendar.
